ntoskrnl/mm/ARM3/section.c

```diff
@@ -1287,6 +1288,9 @@ MiMapViewOfDataSection(IN PCONTROL_AREA ControlArea,
 
     DPRINT("Mapping ARM3 data section\n");
 
+    /* Check for invalid inherit disposition */
+    ASSERT((InheritDisposition >= ViewShare) && (InheritDisposition <= ViewUnmap));
```

Comment:
If you just did what the title of the PR said (i.e., this hunk), this would be an easy merge. Instead you're making lots of changes that are either nitpicks or that people can nitpick about. Do you like long, drawn out, PRs?

Reply:
Removed SAL2 commit.